tests/compiler/test_opcodes.py

```diff
@@ -43,7 +43,7 @@ def test_version_check(evm_version):
 
 def test_get_opcodes(evm_version):
     op = opcodes.get_opcodes()
-    if evm_version in ("paris", "berlin"):
+    if evm_version in ("shanghai", "paris", "berlin"):
```

Comment:
Will it be a good idea to rename `shanghai` with `cancun` already? The EOFv1 will not be added on the **Shanghai** fork and the next candidate for inclusion is **Cancun**.